contracts/ZeroCollateral.sol

```diff
@@ -343,7 +343,19 @@ contract ZeroCollateralMain is ERC20Detailed, ERC20, MinterRole, Chainlink {
         }
     }
 
+    function getCollateralNeeded(uint256 amountBorrow) view public returns(uint256){
+        uint256 poolContributions = (borrowerAccounts[msg.sender].amountPaidRedemptionPool);
```

Comment:
As discussed in a message, it would be better to have `borrowerAddress` as a parameter instead of using `msg.sender`

Reply:
I totally agree. Also, it depends on what you allow users to execute. For example: if you allow a user to get your getCollateralNeeded data, for sure, you need to add a new address parameter (and not use msg.sender). If you don't want to allow that, you need to use the msg.sender. 

We should analyze it and if it is needed to make the changes in a new task.